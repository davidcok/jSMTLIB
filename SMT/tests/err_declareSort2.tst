; declaring an already defined sort
(set-logic QF_UF)
(declare-sort A 0)
(declare-sort A 1)
