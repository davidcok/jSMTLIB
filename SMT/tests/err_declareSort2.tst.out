success
success
(declare-sort A 1)
              ^
(error "The identifier is already declared to be a sort: A")
